### DISCUSSION TITLE
Browser Mode - Is there a way to configure actionTimeout from playwright?

### DISCUSSION THREAD

**Discussion by JoaoCnh** [vitest-dev/vitest#6982] [Q&A] [answered]
In Browser Mode using the Playwright provider, I have a test suite running in CI. From the screenshots I can see the locator found the element correctly but before the click happens it times out because 1000ms is not enough in the slow CI.

Is there a way I can configure the option `actionTimeout` from Playwright?

`launch` and `context` provider options do not have it.

**Comment by sheremet-va** [comment, marked as answer]
You can provide a timeout option directly when calling the method

For now, there is no global configuration that can do that

**Reply by JoaoCnh** [reply]
Gotcha. Are there plans to support this kind of stuff? It seems? simple enough to add, I could try to help out if needed

**Reply by sheremet-va** [reply]
Feel free to open an issue

**Reply by JoaoCnh** [reply]
is there a guideline to opening this sort of stuff? I will read up on it and try to whip something up.

**Reply by sheremet-va** [reply]
Just click "new issue" in the issues tab, there is a form that you can fill there for feature requests
